module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -694,16 +709,6 @@ namespace module::input {
                             // createRotationZ : Mat size [3x3]
                             // Rwt : Mat size [3x3]
```

Comment:
Are these comments still relevant?

Reply:
nope. I have deleted them on the next commit :+1: